src/index.html

```diff
@@ -2,11 +2,348 @@
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link nav__link--is-active is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#laptops"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops&nbsp;&&nbsp;Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="content">
+      <div class="content__catalog catalog">
+        <div class="product-container">
+          <div class="product" data-qa="card">
+            <img
+              class="product__image"
+              src="images/imac.jpeg"
+              alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+            >
+            <p class="product__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </p>
+            <p class="product__serial">Product code: 195434</p>
+            <div class="product__rewiew">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <p class="product__rewiew-title">Reviews: 5</p>
+            </div>
+            <div class="product__price">
+              <p class="product__price-title">Price:</p>
+              <p class="product__price-count">$2,199</p>
+            </div>
+            <a
+              href="#"
+              class="product__btn"
+              data-qa="card-hover"
+            >
+              Buy
+            </a>
+          </div>
+        </div>
+
+        <div class="product-container">
+          <div class="product">
+            <img
+              class="product__image"
+              src="images/imac.jpeg"
+              alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+            >
+            <p class="product__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </p>
+            <p class="product__serial">Product code: 195434</p>
```

Comment:
```add empty lines between multiline sibling blocks of HTML. But don't add empty lines between parent and child elements
            </p>
            
            <p class="product__serial">Product code: 195434</p>
```

Reply:
add empty lines between multiline sibling blocks of HTML. But don't add empty lines between parent and child elements